apps/add_two_ints_service/src/main.c

```diff
@@ -19,9 +19,9 @@ void main(void)
 	RCCHECK(rcl_init_options_init(&options, rcl_get_default_allocator()))
 
 	// Optional RMW configuration 
-	rmw_init_options_t* rmw_options = rcl_init_options_get_rmw_init_options(&options);
-	rmw_uros_options_set_serial_device("1", rmw_options);
-	RCCHECK(rmw_uros_options_set_client_key(0xDEADBEEF, rmw_options))
+	// rmw_init_options_t* rmw_options = rcl_init_options_get_rmw_init_options(&options);
+	// rmw_uros_options_set_serial_device("1", rmw_options);
+	// RCCHECK(rmw_uros_options_set_client_key(0xDEADBEEF, rmw_options))
```

Comment:
Could we remove it?

Reply:
Done

Reply:
But why?
I was just using this for my board that needs to configure another serial device (2).
This makes it harder to use other boards / platforms with it.
I find this info even as comment very valuable.
Maybe remove it from this example but add it somewhere in the documentation. Otherwise you would need to add it in all examples..

Do you want to just give examples that perfectly match your 2 boards or should others be easily learn from them. If they have more of a learning character, it would be suitable to add more information as overhead..

But that's just my personal oppinion

Reply:
Maybe we can add a `rmw_configuration_example` like this one: https://github.com/micro-ROS/micro-ROS-demos/tree/dashing/rcl/configuration_example.

Please open an issue with this requirement @gramss

Reply:
Now we are porting micro-ROS to foxy, so you can contribute with a PR  to `feature/foxy_migration` if you want.